### DISCUSSION TITLE
Use of ansible to deploy teleport configurations

### DISCUSSION THREAD

**Discussion by patrickomatik** [gravitational/teleport#20828] [Q&A]
Does anyone have ansible scripts they can share? I am interested in automatic installation/configuration of teleport ssh and db access on existing servers using ansible to manage that process. (If I don't get lucky, I will share our scripts at later date).

**Comment by webvictim** [comment]
I do have some scripts written to deploy a Teleport service on both and RHEL/Debian-based machines, but they're not in a sufficiently generic state where I could share them at the moment - they have a lot of esoteric things that only apply to my Teleport cluster, etc.

If there's interest, I will consider trying to tidy them up and make them available for public consumption.

**Reply by jackyougit** [reply]
I'm interested in these.

**Comment by juju4** [comment]
Currently 12 roles matching teleport in galaxy
https://galaxy.ansible.com/search?deprecated=false&keywords=teleport&order_by=-relevance&page=1
27 with github search
https://github.com/search?q=ansible+teleport
